packages/paste-core/components/data-grid/src/DataGridCell.tsx

```diff
@@ -23,6 +23,7 @@ export interface DataGridCellProps extends Pick<TdProps, 'textAlign'> {
  * DataGrid cell component. Every visible box in a data grid is powered by the cell.
  *
  * @param {"th" | "td"} as - is it a header or a regular cell
+ * @param {number} colspan - how many columns the cell spans across
```

Comment:
suggestion: probably need to add an old school proptype to the bottom of the file too for the new type restriction, for all the JS users.